repose-aggregator/core/repose-core-api/src/main/java/org/openrepose/core/services/RequestProxyService.java

```diff
@@ -29,19 +29,35 @@
 public interface RequestProxyService {
 
     //TODO: this is the most terrible return value ever
+    //TODO: Deprecate the methods which don't take a connection pool id?
```

Comment:
So we can, but the reason that I haven't done so yet is that I really just want to delete those other methods.

Alas, I don't think I can since this service API is fairly exposed and doing so would force the next release to be a second digit release.

Technically, I also changed some internal DD stuff in a non-backwards compatible way, but those really shouldn't be getting touched by anyone else (e.g., `CacheRequest`). That is, while developers could technically use that code, it isn't exposed as a service interface or anything like that.

So yeah, I'll deprecate those other methods unless someone tells me that we want to delete those methods instead. Or points out a way of doing what I did without changing the contract. ;)